### DISCUSSION TITLE
Support for other boards

### DISCUSSION THREAD

**Discussion by joshk** [toitlang/toit#445] [General]
Hi

First off: really awesome project! 

I've only recently decided to dive into the world of embedded programming, so it was great timing to come across Toit.

I had a question regarding support for other boards. I had been planning to play around with some embedded rust while I build out a little water tank monitoring system. I bought two of these boards: https://www.jaredwolff.com/store/nrf9160-feather/

Do you think support for other boards like these might be possible in the near future?

Thanks!

**Comment by floitsch** [comment]
We are currently focusing on the ESP32 family.
It would be nice to run on other chips as well, but for now concentrating on one chip feels more productive. We rather have a really good solution for the ESP32 than spreading ourselves too thin. Before we look at different chips, we also need to support more variants of the ESP32 family.

That said: eventually we would definitely like to support more chips.

**Reply by joshk** [reply]
Thanks for the reply, and totally understand.
